dev-python/scanpy/scanpy-1.11.3.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=hatchling
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="Single-cell analysis in Python (scalable to >1M cells)"
+HOMEPAGE="https://github.com/scverse/scanpy"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+
+RDEPEND="
+	>=dev-python/anndata-0.9[${PYTHON_USEDEP}]
```

Comment:
https://github.com/scverse/scanpy/blob/7e1ed2f6baed37a2334ccce7557120f20d12633a/pyproject.toml#L49 I think some of the deps are taken from master instead of the 1.11.3 pyprojects.toml, but should not matter for the lower bounds in the end.